packages/server/graphql/types/Organization.ts

```diff
@@ -142,10 +150,10 @@ const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<a
       type: new GraphQLNonNull(OrganizationUserConnection),
       resolve: async ({id: orgId}: {id: string}, _args: unknown, {dataLoader}: GQLContext) => {
         const organizationUsers = await dataLoader.get('organizationUsersByOrgId').load(orgId)
-        organizationUsers.sort((a: OrganizationUser, b: OrganizationUser) =>
+        organizationUsers.sort((a: TOrganizationUser, b: TOrganizationUser) =>
```

Comment:
+1 it looks like we don't need to add the types here or on line 156 below as the types are already known

Reply:
Good find!